Gemfile

```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-gem 'jekyll', '3.3.1'
+gem 'jekyll', '~> 3.3.1'
```

Comment:
We recently updated Jekyll to 3.8.5, you may want to confirm there are no issues with the new plugin with this update

Reply:
Ok yes this is a problem. Jekyll internals have changed from 3.3.1 to 3.8.5.

Yes, this means I will have to update the plugin.